packages/embark-api-client/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "embark-api-client",
+  "version": "4.0.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark api module",
+  "homepage": "https://github.com/embark-framework/embark/tree/master/packages/embark-api-client#readme",
+  "bugs": "https://github.com/embark-framework/embark/issues",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": {
+    "directory": "packages/embark-api-client",
+    "type": "git",
+    "url": "https://github.com/embark-framework/embark.git"
+  },
+  "main": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npm run reset",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint src/",
+    "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
+    "package": "npm pack",
+    "// qa": "npm-run-all lint typecheck build package",
```

Comment:
Are those `//` on purpose?

Reply:
They’re the equivalent of commented lines in a json file (since package.json doesn’t support JSON5).

They were copied over from another package in the monorepo. The idea is that after convertinng to TS they would be uncommented.